plugin/src/main/java/org/jetbrains/research/refactorinsight/folding/ImportFolder.java

```diff
@@ -0,0 +1,114 @@
+package org.jetbrains.research.refactorinsight.folding;
+
+import com.intellij.diff.FrameDiffTool;
+import com.intellij.diff.contents.DocumentContent;
+import com.intellij.diff.fragments.LineFragment;
+import com.intellij.diff.tools.simple.SimpleDiffChange;
+import com.intellij.diff.tools.simple.SimpleDiffViewer;
+import com.intellij.diff.util.Side;
+import com.intellij.openapi.application.ReadAction;
+import com.intellij.openapi.editor.Document;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.editor.FoldRegion;
+import com.intellij.openapi.progress.EmptyProgressIndicator;
+import com.intellij.openapi.progress.ProgressIndicator;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.TextRange;
+import com.intellij.psi.*;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ImportFolder {
+    static Set<FoldRegion> foldRegions;
```

Comment:
Do we need to make it `static`?

Reply:
To avoid creating an object of ImportFolder